src/utils/fluidsynth_priv.h

```diff
@@ -205,6 +205,22 @@ typedef struct _fluid_server_socket_t fluid_server_socket_t;
 typedef struct _fluid_sample_timer_t fluid_sample_timer_t;
 typedef struct _fluid_zone_range_t fluid_zone_range_t;
 
+typedef union _fluid_rvoice_param_t
+{
+    void* ptr;
+    int i;
+    fluid_real_t real;
+} fluid_rvoice_param_t;
+typedef void (*fluid_rvoice_function_t)(void* obj, fluid_rvoice_param_t* param);
+
+enum { EVENT_PARAMS = 6 };
```

Comment:
Maybe call this MAX_EVENT_PARAMS? I think that would make the function signatures a bit clearer, indicating that not all params are used by all event handlers...